pkg/schedule/schedule.go

```diff
@@ -149,12 +150,12 @@ func (f *fifo) run() {
 				f.mu.Unlock()
 				// clean up pending jobs
 				for _, todo := range pendings {
-					todo(f.ctx)
+					f.exec(todo)
```

Comment:
The error should be probably very verbosely printed.

Reply:
Also probably we should count the failed goroutines the same way we have statistics for finished.